slogr/slogr.go

```diff
@@ -21,7 +21,8 @@ limitations under the License.
 // API and of a logr.LogSink through the slog.Handler and thus slog.Logger
 // APIs.
 //
-// Both approaches are currently experimental and need further work.
+// See the README in the top-level [./logr] package for a discussion of
+// interoperability.
```

Comment:
My hope is that pkg.go.dev will render this as a link. I've not tried to verify that beforehand because [it's not that simple anymore](https://mdaverde.com/posts/golang-local-docs/), with `godoc` being deprecated...

Reply:
I can't get pksite to work - it seems to look at github for something, and refuses to include the ./slogr directory

Reply:
I haven't tried at all. I guess we'll have to hope for the best and check after a release.